modules/sb-start-asb.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * applications/service_brokers/installing-ansible-service-broker.adoc
+
+[id='sb-start-asb-{context}']
+= Starting the {asb-name}
+
+Once you have installed the {asb-name} Operator, you can start the {asb-name} using the following procedure.
+
+.Prerequisites
+
+* You have enabled service catalog.
+* You have installed the {asb-name} Operator.
+
+.Procedure
+
+. Navigate in the web console to *Catalog* -> *Installed Operators* and select the `ansible-service-broker` project.
+. Select the *OpenShift Ansible Service Broker Operator*.
+. Under *Provided APIs*, click *Create New* for *OpenShift Ansible Broker*.
```

Comment:
This won't change. Changing the CRD's that the operator watches is not something that we can do right now.

Reply:
@shawn-hurley Okay so this should still just say "Automation Broker"?